tabs/osd.js

```diff
@@ -595,7 +595,8 @@ TABS.osd.initialize = function (callback) {
           MSP.promise(MSPCodes.MSP_OSD_CONFIG)
           .then(function(info) {
             OSD.chooseFields();
-            if (!info.length) {
+            // fc responsed with only 1 field: osd unsupported
+            if (info.length == 1) {
```

Comment:
If the MSP response format used by the firmware was changed at some unknown stage this will potentially break for users of older releases. We should check `if (info.length < 4)` (with 4 being the minimum length of a message that can be decoded properly.

(Also, using strict equality comparisons except for cases where lose equality is explicitly needed makes for better maintainable code.)